server.elm

```diff
@@ -2066,29 +2073,29 @@ lastEditScript = """
           } else {
             onlyGhosts = false;
             sendToUndo(mutation, cur_time);
-            console.log("Attribute is not ghost", mutation);
+            //console.log("Attribute is not ghost", mutation);
```

Comment:
Until we have clever save, please remove the double slash. We need to display when mutations are non ghost to manually record the changes that would go to the `.editor` script.
Same for the three other instances below.